setup_yarn.sh

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 
-set -e
```

Comment:
Any reason for this?

Reply:
it's not used in any other script either (besides entry.sh)
and since i'm not using yarn executable as i planned in the beginning, it's not entirely necessary